src/pages/PassportRedemption/GiveawayPopover.tsx

```diff
@@ -0,0 +1,237 @@
+            <GeneralText>
+              Weekly Giveaway Entries
+            </GeneralText>
+          </InfoText>
+
+          {giveawayData.number_of_visits_left !== 0 ?
+          <InfoText>
+            <TextBold>{giveawayData.number_of_visits_left}</TextBold>
+            <GeneralText>
+              Vendor Tickets away from a Grand Prize Giveaway Entry
+            </GeneralText>
+          </InfoText> :
+          <InfoText icon={true}>
+            <IconWrap>
+              <GreenCheck/>
+            </IconWrap>
+            <GeneralText>Grand Prize Giveaway Entered</GeneralText>
+          </InfoText>
+          }
+        </GiveawayTextContainer>
+        <SubText>
+        Weekly Giveaway winners are announced on Instagram each Monday during the month of September. Grand Prize winners will be announced on October 1st.
+        </SubText>
+        <CloseButton type='button' onClick={() => setShowInfo(false)}>
+          CLOSE
+        </CloseButton>
+      </PopoverContent>}
+      {
+        props.showInstagram ?
+        <InstagramButton onClick={() => setShowInfo(!showInfo)}>
+          <InstagramEnabled/>
+        </InstagramButton> :
+        <InstagramButton disabled={true}>
+          <InstagramDisabled/>
+        </InstagramButton>
+      }
+    </PopoverContainer>
+  )
+}
+
+export default GiveawayPopover
+
+const PopoverContainer = styled.div`
+  position: absolute;
+  right: 0;
+`;
+
+const InstagramButton = styled.div`
+  position: absolute;
+  right: ${(props:IgButtonProps) => props.disabled ? '8.5px' : '4px'};
+  top: ${(props:IgButtonProps) => props.disabled ? '-3px': '1px'};
+  text-decoration: none;
+  color: black;
+  padding: 10px 15px;
+  font-size: 22px;
+`
+
+const PopoverContent = styled.div`
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 340px;
+  height: 315px;
+  right: 12.5px;
+  top: -1px;
+  background: #F2EAE8;
+  box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.25);
+  backdrop-filter: blur(4px);
+  border-radius: 13px;
+  z-index: 2;
+`
+
+const Title = styled.h1`
+  position: absolute;
+  height: 14px;
+  top: 26px;
+  font-family: Open Sans;
+  font-style: italic;
+  font-weight: normal;
+  font-size: 10px;
+  line-height: 14px;
+  display: flex;
+  align-items: center;
+  letter-spacing: 0.15em;
+  color: #000000;
+`
+const Divider = styled.div`
+  position: absolute;
+  width: 237px;
+  top: 52px;
+  margin: 0;
+  border-bottom: 1px solid #8e8e8e;
+`
+
+const GiveawayTextContainer = styled.div`
+  width: 258px;
+  position: absolute;
+  top: 56px;
+  font-family: Open Sans;
+  font-style: normal;
+`
+const IconWrap = styled.span`
+  margin-right: 9px;
+  height: 38px;
+  padding-left: 3px;
+`
+
+const TextBold = styled.span`
+  font-size: 24px;
+  font-weight: bold;
+  font-style: normal;
+  font-family: Open Sans;
+  text-align: left;
+  line-height: 33px;
+  color: black;
+  width: 27px;
```

Comment:
nit: You probably want to remove the width: 27px and add margin-right in `TextBold` or something

![image](https://user-images.githubusercontent.com/37196330/93821780-eda7d980-fc13-11ea-8df3-211d805e9506.png)